EdFi.Buzz.UI.Angular/src/app/Features/StudentDetail/studentDetail.component.html

```diff
@@ -97,3 +97,24 @@ <h2 style="margin-top: 10px;">Siblings: {{student.siblings.length}}</h2>
     </div>
   </div>
 </main>
+
+<!-- Modal -->
+<div class="modal" id="deletenoteconfirmation" data-backdrop="static" data-keyboard="false" tabindex="-1" role="dialog" aria-labelledby="staticBackdropLabel" aria-hidden="true">
+  <div class="modal-dialog">
```

Comment:
You can add the modal-dialog-centered class (to the div with class=modal-dialog) to vertically center the modal

Reply:
Yes, I can do that.